playerbot/strategy/values/CombatTargetsValue.cpp

```diff
@@ -0,0 +1,182 @@
+#include "botpch.h"
+#include "../../playerbot.h"
+#include "CombatTargetsValue.h"
+#include "PossibleTargetsValue.h"
+#include "EnemyPlayerValue.h"
+
+using namespace ai;
+using namespace MaNGOS;
+
+list<ObjectGuid> CombatTargetsValue::Calculate()
```

Comment:
Combat targets is similar to attackers but with some key differences such as no LOS checks and no CC/stunned checks. This was made to prevent the bot from going out of combat when technically there are targets available (but shouldn't be attacked)